salesforce-cdp/1.1/modules/ROOT/pages/index.adoc

```diff
@@ -0,0 +1,62 @@
+= Salesforce CDP Connector 1.1 - Mule 4
+
+Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#anypoint-connectors[Select]
+
+Anypoint Connector for Salesforce CDP (Salesforce CDP Connector) provides Mule users with a quick and easy way to integrate with Salesforce CDP API endpoints. Users can automate data ingestion from third parties, query the CDP Data Lake, search and query customer profile information and calculated insights, and consume data action events to enrich the 360-degree view of their customers in their CDP instance.
+
+Each API call uses a request/response pattern over an HTTPS connection. All required request headers, error handling, and HTTPS connection configurations are built into the connector.
+
+For information about compatibility and fixed issues, refer to the Salesforce CDP Connector release notes.
+
+== Prerequisites
+
+To use this connector, you must be familiar with:
+
+* https://developer.salesforce.com/docs/atlas.en-us.c360a_api.meta/c360a_api/c360a_api_salesforce_cdp_ingestion.htm[The Salesforce CDP Ingestion API]
+* xref:connectors::introduction/introduction-to-anypoint-connectors.adoc[Anypoint Connectors]
+* xref:mule-runtime::whats-new-in-mule.adoc[Mule runtime engine] (Mule)
+* xref:mule-runtime::global-elements.adoc[Elements and global elements] in a Mule flow
+* xref:mule-runtime::mule-app-dev.adoc[How to create a Mule app using Anypoint Studio]
+
+Before creating an app, you must have:
+
+* Credentials to access the Salesforce CDP target resource
+* Anypoint Platform
+* Anypoint Studio version 7.3.x or later
+
+== Common Use Cases For the Connector
+
+These are some common use cases for Salesforce CDP Connector:
+
+* Stream data via insert and delete object endpoints of the Ingestion API Connector
+* Create a bulk job, upload job data, close or abort a job, get a job status, and delete a job as part of the Bulk Ingestion interaction pattern of the Ingestion API Connector
+* Query CDP objects
+* Query Salesforce CDP data lake across data model, lake, unified, and linked objects
```

Comment:
```suggestion
* Query Salesforce the CDP data lake across the data model, lake, unified objects, and linked objects
```

Reply:
It is referring to data model objects, lake objects, unified objects, and linked objects